fail2ban/tests/files/logs/nextcloud-auth

```diff
@@ -0,0 +1,8 @@
+# failJSON: { "time": "2023-09-24T22:34:37.0+02:00", "match": true , "host": "141.30.226.119" }
```

Comment:
Our filter test factory already works in CE(S)T time zone, so normally it would be enough to give the TZ-offset directly in the time.
So for `"time":"2023-09-24T20:34:37+00:00"` it would be `2023-09-24T22:34:37`...
But OK, we can accept this change.

Reply:
I did not know that the test factory works in CET time and assumed it just used the local time of my computer.
In this case I can revert the change to the test factory.